hw/ip/lc_ctrl/doc/lc_ctrl_otp_accessibility.md

```diff
@@ -36,7 +36,7 @@
 </tr>
 <tr>
 <td>RMA_UNLOCK_TOKEN</td>
-<td rowspan="3" colspan="2" style="text-align:center;vertical-align:middle">SECRET0_DIGEST == 0 && CREATOR_SEED_SW_RW_EN == ON</td>
+<td rowspan="3" colspan="2" style="text-align:center;vertical-align:middle">SECRET2_DIGEST == 0 && CREATOR_SEED_SW_RW_EN == ON</td>
```

Comment:
are we missing some other change..? i don't see any other change corresponding to this one.

Reply:
Sorry I should have clarified - this is a typo that became apparent in when we did the review on Monday.
It is not directly related to the ext-clock source change.